device-types/Smartlabs/SML-482-HD-Base.yaml

```diff
@@ -0,0 +1,32 @@
+---
+manufacturer: Smartlabs
+model: SML-482 HD Base
+slug: smartlabs-sml-482-hd-base
+part_number: sml-482-hd-base
+is_full_depth: false
+u_height: 0
+airflow: passive
+front_image: true
+rear_image: true
+subdevice_role: parent
+weight: 140
+weight_unit: g
+interfaces:
+  - name: ether1
+    label: '1'
+    type: 100base-tx
+  - name: usb
```

Comment:
Apart from the network interface the rest are not something we are modeling here.